itest/lnd_zero_conf_test.go

```diff
@@ -239,7 +240,11 @@ func optionScidAliasScenario(ht *lntest.HarnessTest, chantype, private bool) {
 		"--protocol.anchors",
 	}
 
-	bob := ht.Bob
+	bob := ht.NewNode("Bob", nil)
```

Comment:
Hmm not fully sure why we use a new node for bob here when we still close the channel before this function ends? Also given that the open params are also just
```
p := lntest.OpenChannelParams{
	Amt: chanAmt,
}
```

Reply:
Nice observation! Now removed.